_maps/map_files/AlinaStation/job_changes.dm

```diff
@@ -0,0 +1,159 @@
+//custom access for some jobs. pasted together from ministation.
+
+#define JOB_MODIFICATION_MAP_NAME "AlinaStation"
+
+/datum/job/New()
+	..()
+	MAP_JOB_CHECK
+	supervisors = "the captain and the head of personnel"
+
+/datum/outfit/job/New()
+	..()
+	MAP_JOB_CHECK
+	box = /obj/item/storage/box/survival/radio
+
+/datum/job/assistant // Here so assistant appears on the top of the select job list.
```

Comment:
This shouldn't be required, the file inclusion is laid out so that the jobs are defined in standard order before map modifications come in